packages/data-contract/src/actions/registerIdentity.js

```diff
@@ -0,0 +1,14 @@
+require('dotenv').config()
+const { initClient } = require('../utils')
+
+async function createIdentity () {
+  console.log('Creating Identity')
+  const client = initClient()
+
```

Comment:
Why no env checks here?

Reply:
Added mnemonics check to `initClient`

Reply:
initClient should not have this check, we should sanitize everything in the main handler

Reply:
done